client/my-sites/email/inbox/mailbox-selection-list/style.scss

```
@@ -0,0 +1,196 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+body {
+	background-color: var( --studio-white );
+}
+
+.mailbox-selection-list {
+
+	.mailbox-selection-list__header {
+		.formatted-header__title {
+			font-size: $font-title-large;
+			line-height: $font-headline-medium;
+
+			@include break-large {
+				font-size: $font-headline-medium;
+			}
+		}
+
+		.formatted-header__subtitle {
+			font-size: $font-body;
+			line-height: $font-title-medium;
+		}
+	}
+
+	.mailbox-selection-list__new-mailbox-upsell-container {
+		background: #f9f9f9;
+		align-items: center;
+		justify-content: space-between;
+		position: relative;
+		margin: 20px auto 10px;
+		padding: 10px 15px;
+		width: 240px;
+
+		@include break-medium {
+			width: 342px;
+			display: flex;
+			margin-top: 60px;
+			padding: 16px 24px;
+		}
+
+		@include break-large {
+			width: 660px;
+			display: flex;
+			margin-top: 80px;
+			padding: 16px 24px;
+		}
+
+		.mailbox-selection-list__new-mailbox-upsell-messages {
+			font-size: $font-body-small;
+			line-height: 28px;
+
+			h2 {
+				font-weight: bold;
+				font-size: $font-body;
+				line-height: 30px;
+				margin-bottom: 0;
+
+				@include break-medium {
+					margin-bottom: 8px;
+					line-height: 34px;
+				}
+
+				@include break-large {
+					margin-bottom: 8px;
+					line-height: 34px;
+				}
```

Two comments:
1. Are all these `line-height` values strictly necessary? Are our defaults very different?
2. We don't need the `@include break-large` section, as it has the same styles as the `break-medium`, which should remain in force for wider screens.
```suggestion
			line-height: 28px;

			h2 {
				font-weight: bold;
				font-size: $font-body;
				line-height: 30px;
				margin-bottom: 0;

				@include break-medium {
					margin-bottom: 8px;
					line-height: 34px;
				}
```

---

1. Yes. Yes